docker/apollo-server/schemaFetcher.ts

```
@@ -1,12 +1,15 @@
-import { config } from 'dotenv';
+import dotenv, { DotenvConfigOutput } from 'dotenv';
+import dotenvExpand from 'dotenv-expand';
 import { createLogger, Logger, format, transports } from 'winston';
 
-config();
+const env: DotenvConfigOutput = dotenv.config();
 
-const SCHEMA_URL: string = process.env.GRAPHQL_SCHEMA_URL || '';
+dotenvExpand.expand(env);
+
+const SCHEMA_URL: string = process.env.GRAPHQL_SCHEMA_URL!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)